src/ScriptCs.Core/App_Packages/LibLog.3.1/LibLog.cs

```diff
@@ -0,0 +1,1941 @@
+//===============================================================================
```

Comment:
@damianh I was initially against the idea, but as I reflected on it more, and the benefits like you cited I warmed up to it. Common.Logging needed to go, so this is a great replacement.

Thanks!